LondonBritishLibrary/orient/BLorient548.xml

```diff
@@ -31,72 +32,199 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+<locus from="120v"/>
+<title xml:lang="en" type="incomplete" ref="LIT3120Anapho">The anaphora of John Chrysostom</title>
+<textLang mainLang="gez"/>
+<note>Imperfect at the end.</note>
+</msItem>
+
+</msItem>
+
+</msItem>
+
                     </msContents>
-                    
-                    
-                    
-                    
-                    
-                    
+
                     <physDesc>
                         <objectDesc form="Codex">
                             <supportDesc>
                                 <support>
                                     <material key="parchment"/>
+                                    <p>Leaves torn out after <locus target="#f.65">f. 65</locus> and <locus target="f.#69">f. 69</locus>.</p>
                                 </support>
                                 <extent>
                                     <measure unit="leaf">129</measure>
-                                    <dimensions type="outer" unit="mm">
-                                        <height>225</height>
-                                        <width>171</width>
+                                    <dimensions type="outer" unit="in">
+                                        <height>8.875</height>
+                                        <width>6.75</width>
                                     </dimensions>
                                 </extent>
-                                
-                                
                             </supportDesc>
-                            
+
                             <layoutDesc>
                                 <layout columns="2" writtenLines="21"/>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
                                 <desc>The manuscript is written in a good hand, of the <date>19th century</date>; leaves tornout after <locus from="65" to="69"/>; imperfect at the end.
                                 </desc>
-                                
+
                                 <date notBefore="1800" notAfter="1900"/>
-                                
+
                             </handNote>
-                            
-                        </handDesc>   
+
+                        </handDesc>
+                        <decoDesc>
+                          <decoNote xml:id="d1">
+                          <locus target="#7r #31r"/>
+                          <desc>Few attempts at ornament.</desc>
+                          </decoNote>
+                          </decoDesc>
                         <additions>
-                            
+
+
+<list>
+<item xml:id="a1">
+<locus from="2r"/>
+<desc>A quire of six leaves has been prefixed to the volume, containing an additional service of hymns and prayers.</desc>
+</item>
+
```

Comment:
That means it is a composite manuscript, so the whole description should be reworked